api/src/pcapi/connectors/titelive.py

```diff
@@ -27,6 +27,8 @@ def get_jwt_token() -> str:
     def _get_new_jwt_token() -> str:
         try:
             response = requests.post(url, json=payload)
+        except TimeoutError as timeout_error:
+            raise timeout_error
```

Comment:
(détail / style) On n'est pas obligé de récupérer l'exception pour la lever : 

```suggestion
        except TimeoutError:
            raise
```

Reply:
merci pour le code quality tip !